README.md

```diff
@@ -53,7 +59,9 @@ var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
 How would I go about accessing the string "pizza" in the above array?
 
 ```js
-// write code here
+// var foods = ["pizza","fries","hamburger"]
+food[0]
+
```

Comment:
There's an array inside an array above, so `food[0]` would return `["apple", "banana", "strawberry"]`.

You have to use the variables as we define them in these quizzes!